### PR TITLE
Create login and register pages

### PR DESCRIPTION
**Pull request by Amineabali2003**
Create login page and registration page.

Registration page ===> Email, username and password for the attributes to be rendered by the user.

Login ==> Email and password.

### CODE REVIEW

File: front/index.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="h-full bg-white">
```
Comment (idir-44) [inline front/index.html:2]: You should remove that

File: front/src/components/Create_account.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
+export default function Login() {
```
Comment (idir-44) [inline front/src/components/Create_account.jsx:1]: Rename the component to register, and the file Signup

File: front/src/components/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+export default function Login() {
```
Comment (idir-44) [inline front/src/components/Login.jsx:1]: There is a login page on main try using this one

File: front/index.html (hunk 1 above)
Comment (Amineabali2003) [reply front/index.html:2]: I used this class to give a height of 100% and to give a white background to all the html element.

File: front/src/components/Login.jsx (hunk 3 above)
Comment (Amineabali2003) [reply front/src/components/Login.jsx:1]: I didn't understand, you mean use Login.jsx or LoginForm.jsx ?

File: front/src/components/Create_account.jsx (hunk 2 above)
Comment (idir-44) [reply front/src/components/Create_account.jsx]: thanks

**Pull request merged**